server/implementation/pom.xml

```diff
@@ -43,26 +46,30 @@
             <artifactId>jakarta.annotation-api</artifactId>
             <scope>provided</scope>
         </dependency>
-        
+
         <!-- GraphQL Java -->
         <dependency>
             <groupId>com.graphql-java</groupId>
             <artifactId>graphql-java</artifactId>
         </dependency>
-        
+        <dependency>
+            <groupId>com.apollographql.federation</groupId>
+            <artifactId>federation-graphql-java-support</artifactId>
+            <version>2.0.6</version>
```

Comment:
Maybe move the version to parent with `graphql-java` ?

Reply:
done